.github/workflows/release.yml

```diff
@@ -0,0 +1,55 @@
+name: Release
+
+run-name: release new version by @${{ github.actor }}
+
+on:
+  workflow_dispatch:
+
+env:
+  PIP_DISABLE_PIP_VERSION_CHECK: on
+  PIP_NO_CLEAN: on
+  PIP_NO_INPUT: on
+  PIP_PREFER_BINARY: on
+
+permissions:
+  contents: write
+  packages: write
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: false
+
+jobs:
+  containerize:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 1
+
+      - name: Setup python
+        uses: actions/setup-python@v5
+        with:
+          python-version: 3.9
+
+      - name: Install requirements
+        run: make setup
+
+      - name: Install project
+        run: pipenv install --deploy
+
+      - name: Login to container registry
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{github.actor}}
+          password: ${{secrets.GITHUB_TOKEN}}
+
+      - name: Build, tag and push docker image
```

Comment:
This might be more succinct with the `docker/build-push-action` https://github.com/docker/build-push-action

Reply:
good idea yes! i'd like to keep this as is for now though. and then work docker/build-push-action into our [cookiecutter template](https://github.com/robert-koch-institut/mex-template/blob/main/mex-%7B%7B%20cookiecutter.project_name%20%7D%7D/.github/workflows/release.yml) to apply it to all our repos